app/domain/errors.rb

```diff
@@ -307,8 +307,10 @@ module AuthnAzure
           code: "CONJ00053E"
         )
 
-        MissingProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
-          msg:  "Provider fields are missing in xms_mirid {1}",
+        InvalidProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
+          msg:  "Provider fields are in invalid format in xms_mirid {1}." \
+                "xms_mirid must contain the resource provider namespace, the " \
```

Comment:
Question: now that I think about it, is it proper “resource provider namespace”?

Reply:
I saw that somewhere but looking [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types) makes me want to change it to "resource provider"

Reply:
in any case, searching google for "azure resource provider namespace" will lead the customer to the answer. I'll leave it up to you if you want to create a new PR and change it.

Reply:
I am okay with this then. Seems like we could go both ways